.github/workflows/on-pull-request-commit.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
are we not publishing fork-off anywhere?

Reply:
Good point, aded push to ECR workflow